test/context.js

```diff
@@ -18,7 +18,7 @@ describe('context', function() {
 
   it('should support setting max number of sockets', function(done) {
     // 3.2 and above.
-    if (!semver.gte(zmq.version, '3.2.0')) {
+    if (!semver.gte(zmq.version, '3.2.0') || require('os').platform() == 'win32') {
       done();
       return console.warn('Test requires libzmq >= 3.2.0');
```

Comment:
oops I think I just need to drop max sockets under 1023, if so I won't have to fix the message